pallets/parachain-staking/rpc/Cargo.toml

```diff
@@ -0,0 +1,14 @@
+[package]
+authors = ["Interlay <contact@interlay.io>"]
```

Comment:
Interlay is not the author of this. It should rather refer to KILT

Reply:
True. In any case KILT does not have this rpc part, should we put as author pendulum?

Reply:
Sure 👍